main.cpp

```diff
@@ -183,105 +302,264 @@ void Vinyl::moveToneArm(bool is12Inch)
 {
     float distanceToMoveArm;
     distanceToMoveArm = is12Inch ? 6.0f : 3.5f; 
+    std::cout << "Move tone arm " << distanceToMoveArm << "inches" << std::endl;
+}
+
+double Vinyl::bringJoy(int setNumberOfGoodTracks, double setListenerMood)
+{
+    double quality = double(setNumberOfGoodTracks) * setListenerMood;
+    return quality;
+}
+
+double Vinyl::accrueValue(int setnumberOfGoodTracks, int playsOnYoutube)
+{
+    double valueScale = 0.00005;
+    double value = double(setnumberOfGoodTracks) * double(playsOnYoutube);
+    return value * valueScale;
 }
 
 struct Oscillator
 {
-    double sawLevel = 0.0;
-    double squLevel = 1.0;
-    double tuning = 0.0;
-    bool subOscOn = false;
-    int octave = 0; 
+    Oscillator();
+    double sawLevel;
+    double squLevel;
+    double tuning;
+    bool subOscOn;
+    int octave;
 
     double osc1FM(double osc1SawSignal, double osc1SquSignal);
-    double subOscSignal(bool subOscOn);
+    double subOscSignal(bool setDubOscOn);
     int octaveLED(int octave, bool shiftButtonPressed);
 };
 
+Oscillator::Oscillator()
+{
+    sawLevel = 0.0;
+    squLevel = 1.0;
+    tuning = 0.0;
+    subOscOn = false;
+    octave = 0; 
+    std::cout << "Instance of 'Oscillator' has been constructed" << std::endl;
+}
+
 double Oscillator::osc1FM(double osc1SawSignal, double osc1SquSignal)
 {
     return osc1SawSignal * osc1SquSignal;
 }
 
+double Oscillator::subOscSignal(bool setSubOscOn)
+{ 
+    double subsignal = 0.5;
+    setSubOscOn ? subOscOn = 1 : subOscOn = 0;
+    return subsignal * double(subOscOn);
+}
+
 int Oscillator::octaveLED(int selectedOctave, bool shiftButtonPressed)
 {
-    selectedOctave = shiftButtonPressed ? selectedOctave : selectedOctave - 4;
+    selectedOctave = shiftButtonPressed ? selectedOctave -1 : selectedOctave;
     return selectedOctave;
 }
 
 struct LFO
 {
-    double freqLFO = 30;
-    bool squLFO = false;
-    double attenuate = 1.0; 
-    bool speedSlow = true;
-    int routingLFO = 0;
+    LFO();
+    double freqLFO, attenuate;
+    bool squLFO, speedSlow;
+    int routingLFO;
 
     double modCutoff(double attenuate, double LFOSignal);
-    double modPitch(int routingLFO, double LFOSignal);
-    double modSAH(int routingLFO, double LFOSignal);
+    double modPitch(int setRoutingLFO, double setLFOSignal);
+    double modSAH(int setRoutingLFO, double setLFOSignal);
```

Comment:
change these variable names to not contain verbs
Name things what they are.
Name functions based on the action they perform
Variables represent things. 
functions represent actions.
- [ ] revise